qa/L0_infer/infer_test.py

```diff
@@ -100,27 +100,31 @@ def _infer_exact_helper(tester,
                         correlation_id=correlation_id,
                         use_system_shared_memory=TEST_SYSTEM_SHARED_MEMORY,
                         use_cuda_shared_memory=TEST_CUDA_SHARED_MEMORY)
-                # model that supports batching
-                iu.infer_exact(
-                    tester,
-                    pf, (bs,) + tensor_shape,
-                    bs,
-                    input_dtype,
-                    output0_dtype,
-                    output1_dtype,
-                    output0_raw=output0_raw,
-                    output1_raw=output1_raw,
-                    model_version=model_version,
-                    swap=swap,
-                    outputs=outputs,
-                    use_http=use_http,
-                    use_grpc=use_grpc,
-                    use_http_json_tensors=use_http_json_tensors,
-                    skip_request_id_check=skip_request_id_check,
-                    use_streaming=use_streaming,
-                    correlation_id=correlation_id,
-                    use_system_shared_memory=TEST_SYSTEM_SHARED_MEMORY,
-                    use_cuda_shared_memory=TEST_CUDA_SHARED_MEMORY)
+
+                # model that supports batching. Skip for libtorch string I/O
+                if pf == 'libtorch' and tu.validate_for_libtorch_model(
```

Comment:
Has been checked below (search `validate_for_libtorch_model` within the file)

Reply:
Should probably add a new case below to handle Torch string model explicitly, the Torch case just doesn't fit into this helper function.

Reply:
We need to check again with the batch size since string I/O does not support batching.

While the torch case does not fit directly into the helper function it still avoid redundant code. I do not think taking it out into a different helper function is wise.